homeassistant/components/autoskope/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "autoskope",
+  "name": "Autoskope",
+  "codeowners": ["@mcisk"],
+  "config_flow": true,
+  "dependencies": ["http"],
+  "documentation": "https://www.home-assistant.io/integrations/autoskope",
+  "integration_type": "hub",
+  "iot_class": "cloud_polling",
+  "quality_scale": "platinum",
```

Comment:
Let's aim for bronze first, extra is always better, but platinum makes the PR too big